app/src/androidTest/java/org/mozilla/fenix/ui/robots/AddonDetailedMenuRobot.kt

```diff
@@ -0,0 +1,56 @@
+package org.mozilla.fenix.ui.robots
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import org.hamcrest.CoreMatchers.allOf
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.click
+
+class AddonDetailedMenuRobot {
+
```

Comment:
Does this Robot refer to a standalone UI? Or is this a submenu of settings? If so, we should prefix with the "Settingsxxxx"

Reply:
Fixing now to fit standard!